README.md

```diff
@@ -1,5 +1,13 @@
 [![Build Status](https://travis-ci.org/dart-lang/js_facade_gen.svg?branch=master)](https://travis-ci.org/dart-lang/js_facade_gen)
 
+----
+> **Warning**
+
+> # This repository is no longer being maintained and is archived.
+
+> For details on how to write interop interfaces using `package:js`, please visit the [package:js pub page](https://pub.dev/packages/js) or the [Dart documentation for JS interop](https://dart.dev/web/js-interop) (currently in development). The Dart team is working on improving JS interop in the future to make writing such interfaces easier. If there is interest in continuing this repository, feel free to contribute by forking.
```

Comment:
Perhaps drop `'(currently in development)'`? Just given that people will see this note in the future.

Reply:
Sure, I mostly wanted to communicate to users that the mostly-empty page is WIP, but I guess that's a given.